lib/src/manager/render_parameters_manager.dart

```diff
@@ -28,13 +28,21 @@ class RenderParametersManager<T> extends RenderManager<T> {
   /// Add an instance of [RenderObject] by [id].
   @override
   void addRenderObject(T id, RenderObject renderObject) {
-    renderObjects[id] = renderObject as RenderMetricsBox;
+    if (renderObject is! RenderMetricsBox) {
+      return;
+    }
+
+    renderObjects[id] = renderObject;
```

Comment:
I think we have to throw some Exception instead of ignoring of add attempt

Reply:
This is a UI story.
I don't see the benefit of throwing exceptions. It's enough that the worst case will be null.

Reply:
Then it will be like this - you add something, but then you can't find it and you don't know why.

Maybe we just make this:
```dart
void updateRenderObject(T id, RenderMetricsBox renderObject)
```
from this:
```dart
void updateRenderObject(T id, RenderObject renderObject)
```
?